services/catalog/src/simcore_service_catalog/db/repositories/services_shareable.py

```diff
@@ -0,0 +1,106 @@
+import logging
+
+import sqlalchemy as sa
+from models_library.api_schemas_catalog import UserInaccessibleService
+from models_library.services import ServiceKeyVersion
+from models_library.users import GroupID
+from simcore_postgres_database.models.groups import user_to_groups
+from sqlalchemy.sql import and_, or_
+
+from ..tables import services_access_rights
+from ._base import BaseRepository
+
+logger = logging.getLogger(__name__)
+
+
+class ShareableServicesRepository(BaseRepository):
+    """
+    API that operates on services_access_rights and user_to_groups tables
```

Comment:
you are aware that the services repository also does the same right? I mean about the services_access_rights and user_to_groups tables?
later on we should re-divide these 2 repositories.